source/docs/software/can-devices/can-addressing.rst

```diff
@@ -185,3 +185,18 @@ For CAN Nodes to be accepted for use in the FRC System, they must:
 -  Support the minimum Broadcast message requirements as detailed in the Broadcast Messages section.
 -  If controlling actuators, utilize a scheme to assure that the robot is issuing commands, is enabled, and is still present
 -  Provide software library support for LabVIEW, C++, and Java or arrange with *FIRST*\ |reg| or FIRSTs Control System Partners to provide such interfaces.
+
+Universal Heartbeat
+-------------------
+
+The roboRIO provides a universal CAN heartbeat that any device on the bus can listen and react to. This heartbeat is sent every 20ms.
```

Comment:
Are these three meant to be separate paragraphs or part of the same paragraph. They look a bit weird rendered on the docs since it's making them separate.